client/client/app/app.routes.js

```diff
@@ -23,7 +23,17 @@ export default function routes($urlRouterProvider, $httpProvider, $stateProvider
                 collapsedTrackHeaders: null,
                 miew: null,
                 heatmap: null,
-                embedded: null
+                embedded: null,
+
+                closeAllTracks: 'Off',
+                fitAllTracks: 'On',
+                organizeTracks: 'On',
+                genomeAnnotations: null,
+                projectInfo: null,
+                tracksSelection: 'On',
+                hideAll: 'Off',
+                maximise: null,
+                close: null
```

Comment:
- All values should be `null` by default;
- `hideAll` is the same as `embedded`, so we need to pick one;
- all options should also be specified in the url template below.